INSTALL.md

```diff
@@ -84,8 +84,8 @@ Since the [XCSoar](http://www.xcsoar.org/) project already has much of the code
 implemented that is necessary for flight analysis, it makes sense to reuse that
 code where applicable. *SkyLines* is using XCSoar as a python library. This library
 is built and installed by the `xcsoar` python package. To build this library you
-might have to install additional libraries like `libcurl`, which can be installed
-on Debian/Ubuntu by executing `apt-get install libcurl4-openssl-dev`. Please have
+might have to install additional librarie `libcurl` and `freetype`, which can be installed
+on Debian/Ubuntu by executing `apt-get install libcurl4-openssl-dev` and `apt-get install libfreetype6-dev` respectively. Please have
```

Comment:
you can use `apt-get install libcurl4-openssl-dev libfreetype6-dev` to install both dependencies with one command

Reply:
will do, thanks